### ISSUE TITLE
converting this model into .mlmodel

### ISSUE THREAD

**Issue by farazBhatti** [bobqywei/inpainting-partial-conv#12]
Hi, is it possible to run this code on ios platform?

**Comment by greysou1** [comment]
Absolutely! [https://github.com/prisma-ai/torch2coreml](url) This seems to help. I haven't tried it personally but do let me know if anything.

**Comment by farazBhatti** [comment]
link seems to be not working @Prudvi01
